images/base/Dockerfile

```diff
@@ -20,7 +20,7 @@
 # start from ubuntu, this image is reasonably small as a starting point
 # for a kubernetes node image, it doesn't contain much we don't need
 # this stage will install basic files and packages
-ARG BASE_IMAGE=ubuntu:22.04
+ARG BASE_IMAGE=debian:bullseye-slim
```

Comment:
kubernetes's own debian-base image has broken held packages, I'm not sure it would actually make any more sense here. debian:.*-slim is already quite compact and seems well-maintained.

Reply:
I was pleasantly surprised to find that bullseye-slim even has the same iptables version as the existing image, and the build output looks reasonably sized :-)

If we find we need newer packages, we can switch to debian testing track or bullseye backports. It looks like everything in bullseye should be new enough though and this more closely aligns with k8s core images.

Reply:
+1, this also will keep aligned kube-proxy and kindnetd iptables

Reply:
The distro switch didn't quite work out smoothly, https://github.com/kubernetes-sigs/kind/pull/3218